tests/apollo/test_skvbc_preexecution.py

```diff
@@ -366,6 +366,30 @@ async def test_alternate_f_isolated(self, bft_network, tracker):
             last_block = await tracker.get_last_block_id(read_client)
             assert last_block > start_block
 
+    @with_trio
+    @with_bft_network(start_replica_cmd)
+    @verify_linearizability(pre_exec_enabled=True)
+    async def test_conflicting_requests(self, bft_network, tracker):
+        """
+        Launch pre-process conflicting request and make sure that conflicting requests are not committed
+        """
+        bft_network.start_all_replicas()
+
+        read_client = bft_network.random_client()
+        start_block = await tracker.get_last_block_id(read_client)
+
+        ops = 20
+
+        try:
+            with trio.move_on_after(seconds=30):
+                await tracker.run_concurrent_conflict_ops(ops, concurrency_level=2, write_weight=1)
+        except trio.TooSlowError:
+            pass
+
+        last_block = await tracker.get_last_block_id(read_client)
+
+        # We produced at least one conflict.
+        assert last_block < start_block + ops
```

Comment:
As an experiment, can you try running the same test by setting `no_conflicts=True`, and making sure this assert fails?

Reply:
done